vars/openshiftEnsurePushingBinaryBuild.groovy

```diff
@@ -0,0 +1,30 @@
+def call (name, pushUrl) {
```

Comment:
Could this and `vars/openshiftEnsureGenericBinaryBuild.groovy` be one thing, with the value of `spec.output` be a configurable parameter? I'm not sure if that's a desirable thing at all, I just notice that that seems to be the only difference.